Kitodo-LongTimePreservationValidationModule/src/main/java/org/kitodo/longtimepreservationvalidationmodule/KitodoOutputHandler.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.longtimepreservationvalidationmodule;
+
+import edu.harvard.hul.ois.jhove.App;
+import edu.harvard.hul.ois.jhove.HandlerBase;
+import edu.harvard.hul.ois.jhove.Message;
+import edu.harvard.hul.ois.jhove.Module;
+import edu.harvard.hul.ois.jhove.OutputHandler;
+import edu.harvard.hul.ois.jhove.RepInfo;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.UndeclaredThrowableException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import org.kitodo.api.validation.State;
+import org.kitodo.api.validation.ValidationResult;
+
+public class KitodoOutputHandler extends HandlerBase {
+
+    private static final Pattern LINE_SPLITTER = Pattern.compile(System.lineSeparator());
+
+    private static final Constructor<ValidationResult> VALIDATION_RESULT_CONSTRUCTOR;
+
+    static {
+        try {
+            VALIDATION_RESULT_CONSTRUCTOR = ValidationResult.class.getDeclaredConstructor(State.class,
+                Collection.class);
+            VALIDATION_RESULT_CONSTRUCTOR.setAccessible(true);
+        } catch (NoSuchMethodException e) {
+            throw new UndeclaredThrowableException(e);
+        }
+    }
+
+    /**
+     * Method that is executed before the constructor of the parent class.
+     *
+     * @return {@code null}. The value is not used.
+     */
+    private static final String beforeParentConstructor() {
+        Logger.getLogger("edu.harvard.hul.ois.jhove.handler").setLevel(Level.OFF);
```

Comment:
Is switching of an other module logger as a hard coded program instruction a good solution? Why not disabling it through configuration so every user can switch it on if he need it.

Reply:
The program reads its configuration later, so this output is not affected. It only prints "Initializing", which is not helpful for any user.